base/markdown/render/terminal/render.jl

```diff
@@ -61,6 +61,36 @@ function term(io::IO, md::Code, columns)
     end
 end
 
+function term(io::IO, md::Table, columns)
+    col_widths = reduce(max, map(cell -> map(ansi_length, cell), md.rows))
+
+    for (n, row) in enumerate(md.rows)
+        for (i, h) in enumerate(row)
+            a = typeof(md.align) == Symbol ? md.align : md.align[i]
+            print_align(io, h, col_widths[i], a)
+            print(io, " ")
+        end
+        println(io, "")
+
+        if n == 1
+            for w in col_widths
+                print(io, ("-" ^ w) * " ")
+            end
+            println(io, "")
+        end
+    end
+end
+
+function ansi_length(md::Vector{Any})
```

Comment:
I don't really like this overloading – `ansi_length` has a very narrow purpose ("length of a string minus ansi characters") and you're overloading it to mean "string length of a markdown document". The best way to get the string length of some `md` is `length(terminline(md))`.